Makefile

```diff
@@ -183,10 +191,15 @@ test_osdk:
 		OSDK_LOCAL_DEV=1 cargo build && \
 		OSDK_LOCAL_DEV=1 cargo test
 
-.PHONY: initramfs
-initramfs:
+.PHONY: _initramfs
+_initramfs:
 	@$(MAKE) --no-print-directory -C test
 
+.PHONY: initramfs
+initramfs: _initramfs
+	$(eval CARGO_OSDK_INITRAMFS_OPTION := --initramfs=$(shell cat test/build/initramfs_path.txt))
```

Comment:
Adding one extra phony target and exposing a new file under `test` is more complex than it should have been. This Makefile knows whether the gzip will be used or not. So it can simply determine the file suffix directly.
```Makefile
# Skip GZIP to make the creation of initramfs faster
ifeq ($(INITRAMFS_SKIP_GZIP), "1")
INITRAMFS_PATH := test/build/initramfs.cpio
else
INITRAMFS_PATH := test/build/initramfs.cpio.gz
endif

CARGO_OSDK_ARGS += --initramfs=$(INITRAMFS_PATH)
```

Reply:
Thank you for the suggestions! I'll update them when I can, but I have very limited bandwidth at the moment and might be slow to make changes.